test/ac-ansible-roles/install_ac_recipe/tasks/debian.yaml

```diff
@@ -0,0 +1,67 @@
+---
+- name: Create local repo temp folder
+  tempfile:
+    state: directory
+  register: tmp_repo
+  when: local_package_path is defined
+# Local repository https://askubuntu.com/questions/170348/how-to-create-a-local-apt-repository
+- name: Build local repo
+  shell: |
+    apt-get install dpkg-dev -y
+    
+    echo "deb [trusted=yes] file://{{ tmp_repo.path }} ./" >> /etc/apt/sources.list
+    
+    cp {{ local_package_path }}/*.deb {{ tmp_repo.path }}
+    if [ -z "$(ls -A "{{ tmp_repo.path }}")" ]; then
+      echo "No packages where found"
+      exit 1
+    fi
+    
+    cd {{ tmp_repo.path }}
+    dpkg-scanpackages -m . > Packages
+    
+    apt-get update
+  when: local_package_path is defined
```

Comment:
This is nice! At first I din't understand how we were _telling to the recipe_ that it needs to use this repository but I guess we are not: since the version we are trying to install is **not** published in the repository that the recipe is adding, we are using this to install the package.

Would it make sense to document it somehow? I fear we face weird issues if we ever change the version name we are generating here and it collided with an already published package.